util/monitoring.js

```diff
@@ -75,6 +75,30 @@ async function getNetworkReceivedInfo(client, projectId, instanceName, startTime
   return requestAndAssembleInfo(client, request);
 }
 
+async function getMemoryUsageInfo(client, projectId, instanceName, startTime, endTime) {
```

Comment:
Do we not need instanceName for these queries? If so, how about removing the params from `getMemoryUsageInfo` and `getMemoryBytesInfo`?

Reply:
Thanks! done